src/components/RegistrationForm.tsx

```diff
@@ -102,6 +103,18 @@ export default function RegistrationForm({ handles, wardens, className }) {
     }
   };
 
+  useEffect(() => {
+    const link = state.link;
+    const check = new RegExp('^(?:[a-z]+:)?//', 'i');
+    if(link !== undefined){
+      if(!check.test(link) && link !== ''){
+        setHasValidateLinkError(true);
+        return;
+      }
+    }
+    setHasValidateLinkError(false);
+  },[state.link]);
+
```

Comment:
```suggestion
```

I would simply handle is the same way we handle the discord username validation, see here:

```javascript
  const handleChange = useCallback((e) => {
    const { name, value } = e.target;
    setState((prevState) => {
      return { ...prevState, [name]: value };
    });
    if (name === "discordUsername") {
      setIsValidDiscord(discordUsernameRegex.test(value));
    }
    if (name === "username") {
      setisDangerousUsername(value.match(/^[0-9a-zA-Z_\-]+$/) === null);
    }
  }, []);
```